OWNERS.md

```diff
@@ -0,0 +1,15 @@
+# OWNERS
+
+This page lists all maintainers for **this** repository. Each repository in the [Crossplane
+organization](https://github.com/crossplane/) will list their repository maintainers in their own
+`OWNERS.md` file.
+
+Please see the Crossplane
+[GOVERNANCE.md](https://github.com/crossplane/crossplane/blob/master/GOVERNANCE.md) for governance
+guidelines and responsibilities for the steering committee and maintainers.
+
+## Maintainers
```

Comment:
@haarchri has maintainer perms on this repo already, I believe he's intended to be a maintainer as well.  Can you include him in this list also? e.g.,

```
* Christopher Haar <haarchri@upbound.io> ([haarchri](https://github.com/haarchri))
```

Reply:
Absoluteley, no intention to not have him here :)